gammapy/detect/tests/test_lima.py

```diff
@@ -0,0 +1,55 @@
+# Licensed under a 3-clause BSD style license - see LICENSE.rst
+from __future__ import absolute_import, division, print_function, unicode_literals
+
+import numpy as np
+from numpy.testing.utils import assert_allclose, assert_equal
+
+from astropy.convolution import Tophat2DKernel
+from astropy.io import fits
+
+from ...utils.testing import requires_dependency, requires_data
+from ...detect import compute_ts_map, compute_lima_map, compute_lima_on_off_map
+from ...datasets.core import _GammapyExtra
```

Comment:
This is not a good way to access gammapy-extra ... `_GammapyExtra` is an implementation detail.
You should use 
```
from gammapy.datasets import gammapy_extra
filename = gammapy_extra.filename('logo/gammapy_banner.png')
```
of, what I like even better because it saves an import:
```
filename = '$GAMMAPY_EXTRA/logo/gammapy_banner.png'
```